src/ensembl/src/content/app/custom-download/containers/content/filter-accordion/FiltersAccordion.tsx

```diff
@@ -7,32 +7,87 @@ import {
   AccordionItem,
   AccordionItemHeading,
   AccordionItemPanel,
-  AccordionItemButton
+  AccordionItemButton,
+  AccordionItemPermanentBlock
 } from 'src/shared/components/accordion';
 
-import styles from './FiltersAccordion.scss';
-
-import { getFiltersAccordionExpandedPanel } from '../../../state/filters/filtersSelector';
+import { getPreviewResult } from '../../../state/customDownloadSelectors';
+import {
+  getFiltersAccordionExpandedPanels,
+  getSelectedFilters
+} from '../../../state/filters/filtersSelector';
 import {
   setFiltersAccordionExpandedPanel,
   resetSelectedFilters,
   updateSelectedFilters
 } from '../../../state/filters/filtersActions';
 
-import customDownloadStorageService from 'src/content/app/custom-download/services/custom-download-storage-service';
 import JSONValue from 'src/shared/types/JSON';
 
-import { Genes } from './sections';
 import ImageButton, {
   ImageButtonStatus
 } from 'src/shared/components/image-button/ImageButton';
+import BadgedButton from 'src/shared/components/badged-button/BadgedButton';
+import { getCommaSeparatedNumber } from 'src/shared/helpers/numberFormatter';
 import { ReactComponent as ResetIcon } from 'static/img/shared/trash.svg';
+import FiltersAccordionSection from 'src/content/app/custom-download/containers/content/filter-accordion/sections/FiltersAccordionSection';
+import styles from './FiltersAccordion.scss';
+
+type Filter = {
+  [key: string]: boolean;
+};
+type SelectedFilters = {
+  [key: string]: boolean | string | string[] | Filter;
+};
+
+type StateProps = {
+  expandedPanels: string[];
+  selectedFilters: {};
```

Comment:
Why not JSONValue?

Reply:
I remember changing it but it got missed out somehow. It's done now.